packages/node_modules/@ciscospark/test-helper-mocha/src/index.js

```diff
@@ -147,6 +147,34 @@ module.exports = {
     return inNode() ? mochaMethod : noop;
   },
 
+  /**
+   * @param {number} max
+   * @param {string} event
+   * @param {EventEmitter} emitter
+   * @param {function} activityChecker // callback to check if activity matches
+   * @param {string} msg
+   * @returns {Promise<mixed>} Resolves with the activity
+   */
+  expectActivity: function expectActivity(max, event, emitter, activityChecker, msg) {
+    let timer;
+    return Promise.race([
+      new Promise((resolve, reject) => {
+        timer = setTimeout(() => {
+          reject(new Error(`${event} did not fire within ${max}ms${msg ? `: ${msg}` : ''}`));
+        }, max);
+      }),
+      new Promise((resolve) => {
+        emitter.on(event, (result) => {
```

Comment:
take a look at `handleErrorEvent`. using `emitter.on` without unbinding the handler could retain the emitter longer than we want and cause weird things to happen in other tests. `handleErrorEvent` dealt with a similar need but unbinds at the end.

Reply:
👍 added unbind to the end